core/frontend/src/tile/map/MapLayerSources.ts

```diff
@@ -138,20 +138,6 @@ export class MapLayerSources {
     return layers;
   }
 
-  /**
-   *  This function fetch the USG map layer sources. Those are free and publicly available but not worldwide.
-   *  Needs to validate the location of the user before fetching it.
-   */
-  private static async getUSGSSources(): Promise<MapLayerSource[]> {
```

Comment:
Any issue with me removing this? Do we have plans to keep this and use it somewhere?

Reply:
I see those sources as 'nice to have' background data. It should not be part of the core for sure... 

Maybe we could keep it as a key-in defined in the map-layer-extension for now?  So that external applications could turn it ON if needed?

Reply:
Okay, mind if we do that in a separate PR?

Reply:
fine